index.js

```diff
@@ -2,32 +2,41 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var loaderUtils = require("loader-utils"),
-	path = require("path");
-module.exports = function() {};
-module.exports.pitch = function(remainingRequest) {
-	if(this.cacheable) this.cacheable();
-	var query = loaderUtils.getOptions(this) || {};
+
+var path = require("path");
+
+var loaderUtils = require("loader-utils");
+var validateOptions = require('schema-utils');
+
+module.exports = function () {};
+
+module.exports.pitch = function (request) {
+	if (this.cacheable) this.cacheable();
```

Comment:
I left it because it's semver minor & planned to remove with `webpack-defaults` upgrade in the upcoming major, but we can remove it if consensus found about it